index.html

```diff
@@ -210,9 +210,8 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
         </p>
         <p>
           We specify that if the user chooses to capture a [=browser=] [=display-surface=], the user
-          agent MUST instantiate the video track as either {{MediaStreamTrack}} or some sub-class of
-          it. For the purposes of the current document, designate the track's class as
-          <dfn>MediaStreamTrackOrSubclassThereof</dfn>.
+          agent MUST instantiate the video track as a {{MediaStreamTrack}}</p>
```

Comment:
I think you meant `{{BrowserCaptureMediaStreamTrack}}` here?
Otherwise, this paragraph is not necessary, as the UA is already required to instantiate an MST as it is, right?

Reply:
yeah, it might be better to get rid of the paragraph altogether